instrumentation/activity/src/main/java/io/opentelemetry/android/instrumentation/activity/ActivityLifecycleInstrumentation.kt

```diff
@@ -36,7 +37,7 @@ class ActivityLifecycleInstrumentation : AndroidInstrumentation {
     }
 
     private fun buildActivityLifecycleTracer(ctx: InstallationContext): DefaultingActivityLifecycleCallbacks {
-        val visibleScreenService = ctx.serviceManager.getVisibleScreenService()
+        val visibleScreenService = Services.get(ctx.application).visibleScreenTracker
```

Comment:
The static call here is the thing that I think I'm most concerned about in this PR. I think it's manageable in these internal instrumentations (I hope 🤞🏻)....and we can provide overloads or whatever to make it more testable as needed.

Reply:
Definitely, also if there are any issues we can also make breaking changes if needed as it's no longer part of the public api.